### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by Ireneroldan**

### CODE REVIEW

File: staff/irene-roldan/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+```sh
+$mkdir workspace
+```
+
+## cd - cambia el directorio
+
+## mv - (mover archivo a otro nombre) ej mv hola.mundo hola.world
+
+mv hello.world folder-a/folder-b/folder-c
```
Comment (manuelbarzi) [inline staff/irene-roldan/bash/README.md]: enclose this code in back ticks

File: staff/irene-roldan/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# COMMANDS BASH
+
+## pwd - comando para saber la carpeta actual
+
+## ls - Muestra carpetas dentro de una carpeta
+
+$ ls
+
+```sh
+
```
Comment (manuelbarzi) [inline staff/irene-roldan/bash/README.md:15]: add a demo here

File: staff/irene-roldan/bash/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+## touch - crea fichero de texto
+
+## code - Abre en visual studio
+
+## ..q
+
+```js
+
```
Comment (manuelbarzi) [inline staff/irene-roldan/bash/README.md:46]: add a demo here too and in all empty code blocks

File: staff/irene-roldan/bash/README.md
Comment (fercostacastro) [inline staff/irene-roldan/bash/README.md:1]: the explanations might be in English

File: staff/irene-roldan/bash/README.md (hunk 1 above)
Comment (Ireneroldan) [reply staff/irene-roldan/bash/README.md]: DONE!

File: staff/irene-roldan/bash/README.md (hunk 2 above)
Comment (Ireneroldan) [reply staff/irene-roldan/bash/README.md:15]: DONE!

File: staff/irene-roldan/bash/README.md
Comment (Ireneroldan) [reply staff/irene-roldan/bash/README.md:1]: DONE!

File: staff/irene-roldan/bash/README.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+```sh
+(END)..q
+```
+
+## rm -rf - delete directly the folder and all the content
+
+```sh
+$ rm -rf workspace/helloworld
+```
+
```
Comment (Barrufat) [inline staff/irene-roldan/bash/README.md]: Git commands should not be included in this file. If you want, you can create another branch with your git-cheat-sheet file🧐

File: staff/irene-roldan/bash/README.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+/c/Users/Irene/workspace/isdi-bootcamp-202402
+```
+
+## ls - show folders inside another folder
+
+```sh
+$ ls
+README.md  staff/
+
+$ ls -1 - allows you to list files in a single column format
```
Comment (manuelbarzi) [inline staff/irene-roldan/bash/README.md]: move this part to another title with demo code block

File: staff/irene-roldan/bash/README.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+## touch - create text files
+
+```sh
+$ touch readme
+```
+
+## code - open visual studio
+
+## ..q - allow continue writing
```
Comment (manuelbarzi) [inline staff/irene-roldan/bash/README.md]: this is not a bash command (remove it)

File: staff/irene-roldan/bash/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
+# COMMANDS BASH
+
+## pwd - actual folder
+
+```sh
```
Comment (Barrufat) [inline staff/irene-roldan/bash/README.md:5]: Include the commands inside the back ticks of every command section, just as you did in the line 12, for example.

**Pull request merged**